Makefile.images

```diff
@@ -23,12 +23,24 @@ docker_deps = $(shell files=($(1) $$(awk '/COPY/ && substr($$2, 1, 7) != "--from
 package/.image.%: $$(call docker_deps,package/Dockerfile.$$*) $$(call force_image_rebuild,$$*)
 	$(SCRIPTS_DIR)/build_image.sh -i $(lastword $(subst ., ,$@)) -f $< -h $@ $(IMAGES_ARGS)
 
+# Build an OCI tarball from a Dockerfile
+.SECONDEXPANSION:
+package/%.tar: $$(call docker_deps,package/Dockerfile.$$*) $$(call force_image_rebuid,$$*)
+	image=$(firstword $(subst ., ,$(@F))) && \
+	$(SCRIPTS_DIR)/build_image.sh -i $$image -f $< --oci $@ -h package/.image.$$image $(IMAGES_ARGS)
+
 # [images] builds all the container images for the project
 # Default target to build images based on IMAGES variable
 images: $(foreach image,$(IMAGES),package/.image.$(image))
 
+# [multiarch-images] builds all multi-arch container images
+# Default target to build images based on the MULTIARCH_IMAGES variable
+# Default platforms based on the PLATFORMS variable
+multiarch-images: override IMAGES_ARGS += --platform $(PLATFORMS)
+multiarch-images: $(foreach image,$(MULTIARCH_IMAGES),package/$(image).tar)
+
 # [release-images] uploads the project's images to a public repository
-release-images:
+release-images: multiarch-images images
```

Comment:
We have separated the targets on purpose, please see https://github.com/submariner-io/shipyard/blob/devel/gh-actions/release-images/action.yaml

Is there a specific reason to re-couple them?

Reply:
Thanks, this is a left-over from early iterations, I’ll drop it.